pom.xml

```diff
@@ -106,10 +110,6 @@
       <artifactId>vertx-rx-java2</artifactId>
       <optional>true</optional>
     </dependency>
-    <dependency>
-      <groupId>io.vertx</groupId>
-      <artifactId>vertx-core</artifactId>
-    </dependency>
```

Comment:
Why did you move this dependency from here moving it up ?

Reply:
because `vertx-codegen` includes `JsonObject` as well and it's missing the `getValue()` method. That was causing compilation error in my IDE.

It seems that codegen was first in the compilation classpath in my case (not sure if it's always the case)

Reply:
do you want me to keep this commit or not ?

Basically, the good solution would be to make sure that core classes embed in `vertx-codegen` are "compatible" with the real ones.

Reply:
Comparing even with Kafka client poi.xml, the order works well there. It's weird that you needed to invert the order here ... or there is something I'm missing :)

Reply:
:)
The error is only showing up in my IDE (IntelliJ). Maven compilation is not impacted and is successful, whatever ordering is used...
I'm perfectly happy to remove that commit if you wish.

Reply:
Yes please ;)
Then I'll try with my IntelliJ as well.

Reply:
done